custom_metrics/security.js

```diff
@@ -0,0 +1,21 @@
+return JSON.stringify({
+	"iframe-allow": [].map.call(document.querySelectorAll('iframe[allow]'), (x) => {
```

Comment:
For consistency with other metrics could we use the `Array.from` method?
```suggestion
	"iframe-allow": Array.from(document.querySelectorAll('iframe[allow]')).map(iframe => {
```